vaibhav/cart/shopping_cart.rb

```diff
@@ -0,0 +1,128 @@
+#shopping cart 
+
+class Shopping_cart
+
+    def initialize
+        @ary = Array.new{{}}
+        @cart = Array.new{{}}
+    end
+
+    def Add_Products(ary)
+        print 'Enter product name: '
+        productName = gets
+        print 'enter product value: '
+        productValue = gets
+        print 'Enter product quantity: '
+        productQuantity = gets
+        @ary.push([productName, productValue, productQuantity])
```

Comment:
it would be better if you use some logical variable names this makes the code more readable